packages/api/src/errors.js

```diff
@@ -31,11 +46,11 @@ export class UserNotFoundError extends HTTPError {
 }
 UserNotFoundError.CODE = 'ERROR_USER_NOT_FOUND'
 
-export class PinningUnauthorizedError extends HTTPError {
+export class PinningUnauthorizedError extends PinningServiceApiError {
   constructor (msg = 'Pinning not authorized for this user, email support@web3.storage to request authorization.') {
     super(msg, 403)
     this.name = 'PinningUnauthorizedError'
-    this.code = PinningUnauthorizedError.CODE
+    this.reason = PinningUnauthorizedError.CODE
```

Comment:
Shouldn't we refactor this to have `status` and `detail`?

Reply:
Resolved. But only status because detail is set in the constructor.